test/10_isless.jl

```diff
@@ -253,61 +253,61 @@ end
 @testset "comparisons for CategoricalString" begin
     # check that ordering comparisons also fail for CategoricalString
     # (since the AbstractString fallback could break this)
-    pool = CategoricalPool(["a", "b", "c"])
-
-    v1 = CategoricalArrays.catvalue(1, pool)
-    v2 = CategoricalArrays.catvalue(2, pool)
-    v3 = CategoricalArrays.catvalue(3, pool)
-
-    @test_throws ArgumentError v1 < v1
-    @test_throws ArgumentError v1 < v2
-    @test_throws ArgumentError v1 < v3
-    @test_throws ArgumentError v2 < v1
-    @test_throws ArgumentError v2 < v2
-    @test_throws ArgumentError v2 < v3
-    @test_throws ArgumentError v3 < v1
-    @test_throws ArgumentError v3 < v2
-    @test_throws ArgumentError v3 < v3
-
-    @test_throws ArgumentError v1 <= v1
-    @test_throws ArgumentError v1 <= v2
-    @test_throws ArgumentError v1 <= v3
-    @test_throws ArgumentError v2 <= v1
-    @test_throws ArgumentError v2 <= v2
-    @test_throws ArgumentError v2 <= v3
-    @test_throws ArgumentError v3 <= v1
-    @test_throws ArgumentError v3 <= v2
-    @test_throws ArgumentError v3 <= v3
-
-    @test_throws ArgumentError v1 > v1
-    @test_throws ArgumentError v1 > v2
-    @test_throws ArgumentError v1 > v3
-    @test_throws ArgumentError v2 > v1
-    @test_throws ArgumentError v2 > v2
-    @test_throws ArgumentError v2 > v3
-    @test_throws ArgumentError v3 > v1
-    @test_throws ArgumentError v3 > v2
-    @test_throws ArgumentError v3 > v3
-
-    @test_throws ArgumentError v1 >= v1
-    @test_throws ArgumentError v1 >= v2
-    @test_throws ArgumentError v1 >= v3
-    @test_throws ArgumentError v2 >= v1
-    @test_throws ArgumentError v2 >= v2
-    @test_throws ArgumentError v2 >= v3
-    @test_throws ArgumentError v3 >= v1
-    @test_throws ArgumentError v3 >= v2
-    @test_throws ArgumentError v3 >= v3
-
-    @test isless(v1, v1) === false
-    @test isless(v1, v2) === true
-    @test isless(v1, v3) === true
-    @test isless(v2, v1) === false
-    @test isless(v2, v2) === false
-    @test isless(v2, v3) === true
-    @test isless(v3, v1) === false
-    @test isless(v3, v2) === false
-    @test isless(v3, v3) === false
+    pool1 = CategoricalPool(["a", "b", "c"])
```

Comment:
Please avoid renaming variables just to silence warnings, as requested in DataFrames.

Reply:
will revert

Reply:
I guess you can call them `s1` or `cs1`, etc., if you really want to kill those warnings.

Reply:
Sorry, to seem like I am being such a pain just for the sake of these warnings.  I can appreciate that they seem like a triviality, and for any aspect for the package other than updating it that is true.  However, having just been through the process of updating to 0.7, please believe me when I say that they caused me significant confusion.  It can be very difficult to distinguish between intended behavior and bugs unless the tests are clean.  This is coming from someone who's somewhat familiar with DataFrames, someone coming in cold might find it impossible to do such an update, even though the vast majority of the changes that ultimately wound up happening here should only require a working knowledge of core Julia.  It's easy to walk away from tests at the end of the day and forget about the warnings, but every time a PR happens it's a potential source of confusion.

Reply:
Yes, these are rough times. Hopefully this annoying situation won't happen again after 1.0 is released.